tests/bluetooth/bsim_bt/bsim_test_per_adv/src/per_adv_syncer.c

```diff
@@ -307,6 +387,15 @@ static const struct bst_test_instance per_adv_syncer[] = {
 		.test_tick_f = test_tick,
 		.test_main_f = main_per_adv_conn_privacy_syncer
 	},
+	{
+		.test_id = "per_adv_long_data_syncer",
+		.test_descr = "Periodic advertising sync test with maximum "
+			      "data length. Test is used to verify that "
```

Comment:
What does `maximum data length` refer to here? We are not sending 1650 octets, so I guess it refers to the maximum data in a single LTV (bt_data) struct? If so, that can be done with just a single bt_data struct, but you are using two, so the description here doesn't seem to match the actual test case implementation